values.yaml

```diff
@@ -411,12 +413,14 @@ client:
   # also changes the clients' advertised IP to the hostIP rather than podIP.
   exposeGossipPorts: false
 
-  # Resource requests, limits, etc. for the client cluster placement. This
-  # should map directly to the value of the resources field for a PodSpec.
-  # By default no direct resource request is made.
-  # NOTE: The use of a YAML string is deprecated. Instead, set directly as a
-  #       YAML map.
```

Comment:
Should we keep the comment about YAML string deprecation?

Reply:
yes, rebase mistake